pkg/webhooks/podimagespec/podimagespec.go

```diff
@@ -0,0 +1,274 @@
+package podimagespec
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/k8sutil"
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+
+	configv1 "github.com/openshift/api/operator/v1"
```

Comment:
wrong package for configv1 https://github.com/openshift/api/blob/master/imageregistry/v1/register.go